common/app/experiments/Experiments.scala

```diff
@@ -64,3 +65,12 @@ object OfferHttp3
       sellByDate = LocalDate.of(2024, 1, 3),
       participationGroup = Perc1E,
     )
+
+object DisableAffiliateLinks
+    extends Experiment(
+      name = "disable-affiliate-links",
+      description = "Disable affiliate links for 5% of users so that we can measure differences in engagement",
+      owners = Seq(Owner.withEmail("commercial.dev@theguardian.com")),
+      sellByDate = LocalDate.of(2024, 1, 30),
+      participationGroup = Perc5A,
```

Comment:
Can we maybe go with a 2% test instead of 5%? Only because it'll be running for a while, and there's only one 5% bucket, which someone else might want to use.

Reply:
I've set the test to 2%